envs/postgres-s3-spark/.whirl.env

```diff
@@ -13,9 +13,10 @@ POSTGRES_USER=postgres
 POSTGRES_DB=postgresdb
 
 # Spark variables
-SPARK_VERSION=2.4.1
+SPARK_VERSION=3.0.1
```

Comment:
somewhere else you're referring to 3.0, perhaps we can be consistent and state the same version everywhere?

Reply:
Yes. The other defined version relates to the tag of a docker image. There is no 3.0.1 tag unfortunately at the moment.